stable/atlantis/templates/service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "atlantis.fullname" . }}
+  labels:
+    app: {{ template "atlantis.name" . }}
+    chart: {{ template "atlantis.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  {{- with .Values.service.annotations }}
+    annotations:
+  {{ toYaml . | indent 4 }}
```

Comment:
this needs to have no indentation and should be 6

Reply:
sorry 4 is fine, but the toYaml block needs to be left aligned.  If you have more then one annotation it would currently error.

Reply:
Fixed.  It's actually `annotations` that was at the wrong indentation level.